src/main/java/com/linecorp/armeria/client/circuitbreaker/FailFastException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.circuitbreaker;
+
+/**
+ * An exception indicating that a request has been failed by circuit breaker.
+ */
+public final class FailFastException extends RuntimeException {
```

Comment:
Given that this exception is gonna be instantiated a lot when the circuit is broken, would it be a good idea to override `fillStackTrace()` so its instantiation cost is very small?